fe/fe-core/src/main/java/com/starrocks/common/util/KafkaUtil.java

```diff
@@ -178,11 +179,17 @@ private PProxyResult sendProxyRequest(PProxyRequest request) throws UserExceptio
             TNetworkAddress address = new TNetworkAddress();
             try {
                 List<Long> backendIds = GlobalStateMgr.getCurrentSystemInfo().getBackendIds(true);
+                // TODO: need to refactor after be split into cn + dn
+                if ((RunMode.getCurrentRunMode() == RunMode.SHARED_DATA)) {
```

Comment:
whether should change to get list from warehouse cluster?

Reply:
not necessary